### ISSUE TITLE
No Link to Retrospect Templates

### ISSUE THREAD

**Issue by icguy** [umbraco/UmbracoDocs#2498]
***This is the page with issues: https://github.com/umbraco/UmbracoDocs/edit/master/Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md***

Problems summary:

1.  Provide link to Retrospect template in tutorial
2. Retrospect template has invalid Razor code in index.html

Details:
1. The tutorial says "Open up "index.html" from the "Retrospect" template in your favourite text editor". The user clearly doesn't have this on their computer because the tutorial specifically asked them to perform a clean installation instead of the one with the Retrospect template. The tutorial should provide a means for the user to reach the template without having to google it.

2. The retrospect template has a twitter handle ("@templatedco") in the copyright comment which results in invalid Razor code. The tutorial should either instruct the user to remove it or choose another template.

**Comment by sofietoft** [comment]
Thanks for the report @icguy !

The tutorial you're referring to here, actually stretches over multiple articles.
The very first article in the guide, is this one: https://our.umbraco.com/documentation/Tutorials/Creating-Basic-Site/Getting-Started/

In there is a section about "[What you will need](https://our.umbraco.com/documentation/Tutorials/Creating-Basic-Site/Getting-Started/#what-youll-need)" in order to follow the tutorial.

To avoid confusion, perhaps we could link to that section in [the article you're referring to](https://our.umbraco.com/documentation/Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/) as well?
What do you think of that? Would that solve the confusion here? 🤔

**Comment by icguy** [comment]
@sofietoft Oh shoot, i missed the "What you will need" section. Apparently reading tutorials thoroughly from start to finish isn't my forte. My bad. Maybe linking to the other article might be helpful, but isn't strictly necessary. Users (including me) need to RTFM after all. So do with it what you wish. Did you check my other point though?

**Comment by sofietoft** [comment]
No worries 😄 We don't really have a nice way to handle these multi-page guides and articles, so it can be a bit confusing to keep track of it all.

Ah, sorry! Seems I was too focused on the other things... it's the `@` sign in the Twitter handle that's messing up the Razor. Would be a good idea to add a note about that to the article 🤔 

[Our docs are open source](https://our.umbraco.com/documentation/Contribute/) - would you be up for adding the link and the note? 😄 
I'll add the up-for-grabs label on this one for now 💪

**Comment by umbrabot** [comment]
Hi @icguy,

We're writing to let you know that we would love some help with this issue. We feel that this issue is ideal to flag for a community member to work on it. Once flagged here, folk looking for issues to work on will know to look at yours. Of course, please feel free work on this yourself ;-).  If there are any changes to this status, we'll be sure to let you know. 

For more information about issues and states, have a look at [this blog post](https://umbraco.com/blog/the-umbraco-issue-tracker-process/)

Thanks muchly, from your friendly Umbraco GitHub bot :-)

**Comment by sofietoft** [comment]
Apologies for the long wait on this one!

Looks like a note about how the `@` should be removed from the template 💪 
With that, I'll close this issue for now.

Please feel free to create a new issue or re-open this one, should some unanswered questions remain.
Thanks!
